wallet/hardware.md

```diff
@@ -82,6 +83,31 @@ This type of set-up does come with additional hurdles that may be more difficult
 
 ***
 
+## Jade
+
+The Blockstream [Jade](https://blockstream.com/jade/) is one of the smallest hardware wallets on the market but offers a lot of features like the support of SeedQR and a built-in battery. Beginners or people who are not interested in the us of stateless devices have the option to use USB, Bluetooth or SeedQR with a stored seed in combination with a virtual secure element (pin server). There is also the possibility to use the Jade as stateless signing device only. 
+
+<p align="center">
+<img src="/assets/img/jade.png" class=responsive width="350" height="350" maxheight="350">
```

Comment:
Can you get an image with the background removed so it matches the style of the page please?

I like to use a tool like https://www.remove.bg/

Reply:
done with [e9ac1b3](https://github.com/BitcoinQnA/Bitcoiner.Guide/pull/39/commits/e9ac1b39209d9705dd6aa0935799409c910798ea)